core/src/main/scala-2.13+/com.softwaremill.diffx/package.scala

```diff
@@ -0,0 +1,56 @@
+package com.softwaremill
+
+import scala.annotation.compileTimeOnly
+import scala.collection.Factory
+
+package object diffx {
+  def red(s: String): String = Console.RED + s + Console.RESET
```

Comment:
maybe extend a 2.12 & 2.13-shared trait for the things that are the same for both scalas?